src/modules/fancyzones/FancyZonesLib/DraggingState.h

```diff
@@ -18,9 +19,12 @@ class DraggingState
 
 private:
     void OnMouseDown();
+    void OnMiddleMouseDown();
 
     std::atomic<bool> m_mouseState;
+    std::atomic<bool> m_middleMouseState;
     SecondaryMouseButtonsHook m_mouseHook;
+    MiddleMouseButtonsHook m_middleMouseHook;
```

Comment:
I believe it'd be better to make `SecondaryMouseButtonHook` just `MouseButtonHook` and add there different callbacks for right-click and middle-click.

Reply:
This is resolved now.